src/test/java/org/candlepin/subscriptions/event/EventControllerTest.java

```diff
@@ -185,12 +185,12 @@ void setup() {
   @Test
   void testPersistServiceInstances_WhenValidPayload() {
 
-    Set<String> eventRecords = new HashSet<>();
+    List<String> eventRecords = new ArrayList<>();
     eventRecords.add(eventRecord1);
     eventRecords.add(eventRecord2);
     eventRecords.add(eventRecord3);
     eventController.persistServiceInstances(eventRecords);
-    verify(optInController, times(2)).optInByOrgId(any(), any());
+    verify(optInController, times(3)).optInByOrgId(any(), any());
```

Comment:
this is a result of changing Set to List. The duplicates were getting removed earlier in the process so opt in was called one less time.